src/custom/hooks/useApproveCallback/useApproveCallbackMod.ts

```diff
@@ -10,6 +10,9 @@ import { calculateGasMargin } from 'utils/calculateGasMargin'
 import { useTokenContract } from 'hooks/useContract'
 import { useActiveWeb3React } from 'hooks/web3'
 import { useTokenAllowance } from 'hooks/useTokenAllowance'
+import { ethers } from 'ethers'
+
+const APPROVE_GAS_LIMIT_DEFAULT = ethers.BigNumber.from('150000')
```

Comment:
nit: consider leaving a comment for why 150.000 is the default

Reply:
Hmmm, I did get that error and the default was set 
![screenshot_2021-11-04_11-03-25](https://user-images.githubusercontent.com/43217/140397179-9d8cdb6c-6679-4ee5-9053-213285de0948.png)

But the wallet did not receive/did not use this value...